framework/test/jest/specs/unit/controller/channels/in_memory_channel.spec.js

```diff
@@ -75,21 +73,21 @@ describe('InMemoryChannel Channel', () => {
 	describe('#constructor', () => {
 		it('should create the instance with given arguments.', () => {
 			// Assert
-			expect(inMemoryChannel.bus).toBe(params.bus);
+			expect(inMemoryChannel.bus).toBe(undefined);
```

Comment:
this can be replaced with `toHaveProperty` matcher.

Reply:
Can you give an example, pls? I've changed to `undefined` because we not setting the bus in the constructor anymore.

Reply:
Ah in that case please move this test from constructor to registerToBus suite.

Also just fyi I meant `expect(inMemoryChannel).toHaveProperty('bus')` could be used if the property is declared but was not assigned yet.